cmd/README.md

```diff
@@ -0,0 +1,738 @@
+            "minBlockGasCost": 0,
+            "maxBlockGasCost": 10000000,
+            "targetBlockRate": 2,
+            "blockGasCostStep": 500000
+        },
+        "helloWorldConfig": {
+            "blockTimestamp": 0
+        }
+    },
+    "alloc": {
+        "8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC": {
+            "balance": "0x52B7D2DCC80CD2E4000000"
+        },
+        "0x0Fa8EA536Be85F32724D57A37758761B86416123": {
+            "balance": "0x52B7D2DCC80CD2E4000000"
+        }
+    },
+    "nonce": "0x0",
+    "timestamp": "0x0",
+    "extraData": "0x00",
+    "gasLimit": "0x1312D00",
+    "difficulty": "0x0",
+    "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "coinbase": "0x0000000000000000000000000000000000000000",
+    "number": "0x0",
+    "gasUsed": "0x0",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+}
+```
+
+Adding this to our genesis enables our HelloWorld precompile.  
+
+``` json
+"helloWorldConfig": {
+  "blockTimestamp": 0
+},
+```
+
+As a reminder, we defined `helloWorldConfig` in `./params/precompile_config.go`. By putting this in genesis, we enable our HelloWorld precompile at blockTimestamp 0. 
+
+``` go
+// PrecompileUpgrade is a helper struct embedded in UpgradeConfig, representing
+// each of the possible stateful precompile types that can be activated
+// as a network upgrade.
+type PrecompileUpgrade struct {
+	ContractDeployerAllowListConfig *precompile.ContractDeployerAllowListConfig `json:"contractDeployerAllowListConfig,omitempty"` // Config for the contract deployer allow list precompile
+	ContractNativeMinterConfig      *precompile.ContractNativeMinterConfig      `json:"contractNativeMinterConfig,omitempty"`      // Config for the native minter precompile
+	TxAllowListConfig               *precompile.TxAllowListConfig               `json:"txAllowListConfig,omitempty"`               // Config for the tx allow list precompile
+	FeeManagerConfig                *precompile.FeeConfigManagerConfig          `json:"feeManagerConfig,omitempty"`                // Config for the fee manager precompile
+	HelloWorldConfig                *precompile.HelloWorldConfig                `json:"helloWorldConfig,omitempty"`
+	// ADD YOUR PRECOMPILE HERE
+	// {YourPrecompile}Config  *precompile.{YourPrecompile}Config `json:"{yourPrecompile}Config,omitempty"`
+}
+```
+
+Now we can get the network up and running. 
+Open some terminal tabs and enter the following commands.
+
+Here's some more information on [avalanche-network-runner](https://docs.avax.network/subnets/network-runner) and how to download it. 
+
+``` go
+// Start the server 
+avalanche-network-runner server \
+--log-level debug \
+--port=":8080" \
+--grpc-gateway-port=":8081"
+
+// In the root of the repo, run this to get the latest subnetevm binary 
+./scripts/build.sh
+
+// Set paths
+export AVALANCHEGO_EXEC_PATH="${HOME}/go/src/avalanchego/build/avalanchego"
```

Comment:
I think the better version of this would be to use `GOPATH/src/...` instead of assuming the user has set up their `GOPATH` directly within the home directory.

Reply:
Actually this is missing `github.com/ava-labs` before `avalanchego`.

Reply:
ah thanks.